src/main/scala/com/amazon/deequ/analyzers/KLLSketch.scala

```diff
@@ -0,0 +1,122 @@
+package com.amazon.deequ.analyzers
+
+import com.amazon.deequ.analyzers.catalyst.KLLSketchSerializer
+import com.amazon.deequ.analyzers.runners.IllegalAnalyzerParameterException
+import com.amazon.deequ.metrics.{BucketDistribution, BucketValue, KLLMetric}
+import scala.collection.mutable.ListBuffer
+import scala.util.Try
+import java.nio.ByteBuffer
+import com.amazon.deequ.analyzers.Analyzers._
+import com.amazon.deequ.analyzers.Preconditions.{hasColumn, isNumeric}
+import com.amazon.deequ.analyzers.runners.MetricCalculationException
+import org.apache.spark.sql.DeequFunctions.stateful_kll
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.{Column, Row}
+
+import scala.util.Failure
+
+/** State for KLL Sketches */
+case class KLLState(qSketch: QuantileNonSample[Double], globalMax: Double, globalMin: Double)
+  extends State[KLLState] {
+
+  /** Add up states by merging sketches */
+  override def sum(other: KLLState): KLLState = {
+    val mergedSketch = qSketch.merge(other.qSketch)
+    KLLState(mergedSketch, Math.max(globalMax,other.globalMax), Math.min(globalMin,other.globalMin))
+  }
+}
+
+object KLLState{
+  // from Array[Byte] to original type
```

Comment:
Please use /**/ comments starting with capital letters

Reply:
Sure. Add a /**/ comments on the fromBytes function.